src/Parser.zig

```diff
@@ -4357,7 +4357,7 @@ const CallExpr = union(enum) {
         return switch (self) {
             .standard => true,
             .builtin => |builtin| switch (builtin.tag) {
-                .__builtin_va_start, .__va_start, .va_start => arg_idx != 1,
+                .__builtin_va_start, .__va_start => arg_idx != 1,
```

Comment:
What's this change?

Reply:
Oops, thank you for catching that. I initially updated `BuiltinFunction.zig` by regenerating it with the python script, against LLVM16 (I believe it was initially generated with LLVM15). But they changed how they represent `LIBBUILTIN` headers, and the script failed to render those. Before I realized what had happened I updated that switch, but then I figured it out and reverted the `BuiltinFunction.zig` change and just made the decls pub, planning to do the update as a separate commit. But then I forgot to revert the parser change. I've pushed an update that removes that change.